client/me/developer/features/style.scss

```
@@ -0,0 +1,51 @@
+@import "@wordpress/base-styles/breakpoints.scss";
+@import "@automattic/typography/styles/variables";
+
+.developer-features-list {
+	display: grid;
+	grid-template-columns: repeat(3, 1fr);
+	grid-gap: 24px;
+	padding-bottom: 24px;
+	font-family: "SF Pro Text", $sans;
+
+	.developer-features-list__item {
+		display: flex;
+		flex-direction: column;
+		margin-bottom: 0;
+
+		a {
+			font-size: rem(13px);
+			color: #c9356e;
```

I still think this needs updating to work on other colour schemes. 

```suggestion
			color: var(--color-accent);
```

---

I have already updated it